batchflow/models/eager_torch/resnet.py

```diff
@@ -53,6 +56,8 @@ def __init__(self, inputs=None, layout='cnacna', filters='same', kernel_size=3,
             groups = [1] + groups + [1]
             bottleneck = 4 if bottleneck is True else bottleneck
             filters = [filters[0]] + filters + [filters[0] * bottleneck]
+        if se:
```

Comment:
@Dimonovez `se` is not needed. Use `layout` only

Reply:
If you do not use `se`, there will be problems with the` bottleneck`.
Also, more complex parameters will need to be pass in ResBlock.